src/Grid/grid_generators.jl

```diff
@@ -69,7 +69,7 @@ end
 
 function _generate_2d_nodes!(nodes, nx, ny, LL, LR, UR, UL)
```

Comment:
```suggestion
function _generate_2d_nodes!(nodes::Vector{Node{2, T}}, nx, ny, LL, LR, UR, UL) where T
```

Reply:
I agree with this solution, taking the type from node is more descriptive than using left